pkg/gitreceive/storage/slug_builder_info.go

```diff
@@ -0,0 +1,29 @@
+package storage
+
+import (
+	"fmt"
+
+	"github.com/deis/builder/pkg/gitreceive/git"
+)
+
+// SlugBuilderInfo contains all of the object storage related information needed to pass to a slug builder
+type SlugBuilderInfo struct {
+	PushKey string
+	PushURL string
+	TarKey  string
+	TarURL  string
+}
+
+// NewSlugBuilderInfo creates and populates a new SlugBuilderInfo based on the given data
+func NewSlugBuilderInfo(s3Endpoint, appName, slugName string, gitSha *git.SHA) *SlugBuilderInfo {
+	tarKey := fmt.Sprintf("home/%s/tar", slugName)
+	// this is where workflow tells slugrunner to download the slug from, so we have to tell slugbuilder to upload it to here
+	pushKey := fmt.Sprintf("home/%s:git-%s/push", appName, gitSha.Short)
```

Comment:
Fixed in https://github.com/arschles/builder/commit/1e4f08f0eae7c0efecefa4047b3ecd8fb5af72b4